build.gradle

```diff
@@ -299,7 +299,7 @@ dependencies {
     implementation 'org.bouncycastle:bcprov-jdk15on:1.70'
     implementation 'org.bouncycastle:bcpkix-jdk15on:1.70'
     implementation "org.opensearch:performanceanalyzer-rca:${version}"
-    implementation "org.opensearch:performanceanalyzer-commons:${version}"
+    implementation "org.opensearch:performance-analyzer-commons:1.0.0-SNAPSHOT"
```

Comment:
`1.0.0-SNAPSHOT` is meant for tracking active development and we use the release version otherwise. 

Let us fix this in follow-up PR.